ykcompile/src/lib.rs

```diff
@@ -49,25 +49,28 @@ impl Display for CompileError {
 }
 
 /// Converts a register number into it's string name.
-fn reg_num_to_name(r: u8) -> &'static str {
-    match r {
-        0 => "rax",
-        1 => "rcx",
-        2 => "rdx",
-        3 => "rbx",
-        4 => "rsp",
-        5 => "rbp",
-        6 => "rsi",
-        7 => "rdi",
-        8 => "r8",
-        9 => "r9",
-        10 => "r10",
-        11 => "r11",
-        12 => "r12",
-        13 => "r13",
-        14 => "r14",
-        15 => "r15",
-        _ => unimplemented!(),
+fn reg_num_to_name(loc: &Location) -> &'static str {
```

Comment:
Shouldn't this really be named differently now?

Reply:
Oh, has this been merged already? I'll probably to that in a separate PR though.

Reply:
I mean, the name of the function `reg_num_to_name` is confusing, because it no longer takes a register number. How about `local_to_reg_name`?

Reply:
Fixed in 20600b7